src/OSPSuite.UI/Extensions/ChartControlExtensions.cs

```diff
@@ -30,5 +36,97 @@ public static void SetFontAndSizeSettings(this ChartControl chartControl, ChartF
             xyDiagram.GetAllAxesY().Each(axis => axis.Title.Font = fontAxis);
          }
       }
+
+      private const string WATERMARK_ANNOTATION = "WATERMARK_ANNOTATION";
+
+      /// <summary>
+      ///    Copies the chart to clipboard as an image using export settings if defined
+      ///    Otherwise uses current visual settings
+      /// </summary>
+      public static void CopyToClipboard(this UxChartControl chartControl, IChart chart, string watermark)
+      {
+         using (var cloneOfChartControl = (ChartControl)chartControl.Clone())
+         {
+            cloneOfChartControl.SetFontAndSizeSettings(chart.FontAndSize, chartControl.Size);
+            if (chart.IncludeOriginData)
+               AddOriginData(cloneOfChartControl, chart);
+
+            if (!string.IsNullOrEmpty(watermark))
+               AddWatermark(cloneOfChartControl, chart, watermark);
+
+            chartControl.CopyChartToClipboard(cloneOfChartControl);
+         }
+      }
+
+      public static void AddWatermark(this ChartControl chartControl, IChart chart, string watermark)
+      {
+         var shouldHideWatermark = string.IsNullOrEmpty(watermark);
+         var watermarkAnnotation = chartControl.Annotations.OfType<TextAnnotation>().FirstOrDefault(x => Equals(x.Name, WATERMARK_ANNOTATION));
+         //we add a watermark but this should be removed
+         if (watermarkAnnotation != null && shouldHideWatermark)
+            chartControl.Annotations.Remove(watermarkAnnotation);
+
+         if (shouldHideWatermark)
+            return;
+
+         if (watermarkAnnotation == null)
+            watermarkAnnotation = createWatermarkAnnotation(chartControl);
+
+         watermarkAnnotation.Font = chart.FontAndSize.FontFor(x => x.WatermkarkSize);
+         watermarkAnnotation.Text = watermark.InBold();
+         updateAnnotationPosition(watermarkAnnotation, chartControl);
+      }
+
+      private static TextAnnotation createWatermarkAnnotation(ChartControl chartControl)
+      {
+         var annotation = chartControl.Annotations.AddTextAnnotation(WATERMARK_ANNOTATION);
+         annotation.ShapePosition = new FreePosition();
+         annotation.Angle = -45;
+         annotation.ConnectorStyle = AnnotationConnectorStyle.None;
+         annotation.BackColor = Color.Transparent;
+         annotation.Border.Visibility = DefaultBoolean.False;
+         annotation.TextColor = Constants.ChartFontOptions.DEFAULT_FONT_COLOR_WATERMARK;
```

Comment:
We do not have settigns for FONT COLOR Anywhere. I did not want to introduce somethign just foi a single font. In theory, we could also define FONT_COLOR in the export settings as part of the UI. Nice to have for another time